src/MICore/LaunchOptions.cs

```diff
@@ -248,7 +351,7 @@ public static ReadOnlyCollection<SourceMapEntry> CreateCollection(Xml.LaunchOpti
 
         public static ReadOnlyCollection<SourceMapEntry> CreateCollection(Dictionary<string, string> source)
         {
-            IList<SourceMapEntry> sourceMaps = source?.Select(x => new SourceMapEntry() { EditorPath = x.Value, CompileTimePath = x.Key }).ToList();
+            IList<SourceMapEntry> sourceMaps = source?.Select(x => new SourceMapEntry() { EditorPath = x.Key, CompileTimePath = x.Value, UseForBreakpoints = true }).ToList();
```

Comment:
>UseForBreakpoints [](start = 137, length = 17)

I would suggest adding a // TODO comment and some sort of work item to support setting this to false as there are some cases where one wouldn't want that on.